spec/features/invite_existing_user_spec.rb

```diff
@@ -12,6 +12,6 @@
     fill_in "Email", with: "existing@example.com"
     #click button: send an invitation
     find('input[name="commit"]').click
-    expect(page).to have_content "existing@example.com has been invited to join your team"
+    expect(page).to have_content "#{user.first_name} has been invited to join your team"
```

Comment:
@FrankCardillo yes I did!

I think to avoid merging issues, you can leave it as spec/features/invite_existing_user_spec.rb.  We can merge in your work to master,then I can add it to the user feature spec (spec/features/user_spec.rb) on my PR, then merge mine in after.  Does that make sense?